packages/capabilities/src/space.js

```diff
@@ -72,6 +72,9 @@ export const allocate = capability({
 export const contentServe = capability({
   can: 'space/content/serve/*',
   with: SpaceDID,
+  nb: Schema.struct({
+    authToken: Schema.string().optional(),
```

Comment:
**issue:** This should be `token`, not `authToken`. It also isn't optional: it has to be `string | null`. We want people to explicitly opt in to public access.

**thought:** Which is easy enough to fix, but also has me concerned that it's _way_ too easy for wildcard capability definitions to be out of sync with their children. Not sure there's much we can do about that at the moment, though, at a fundamental level.

**suggestion:** Could we at least abstract the `token` part of the schema to something shared between the two? That means moving the transport one over here, which makes sense; they should be in the same place, wherever that is.